src/lib.rs

```diff
@@ -44,14 +44,73 @@ fn unsafe_alloc_buffer(len: usize) -> Vec<u8> {
     ret
 }
 
+#[derive(Copy, Clone)]
+pub struct Uint128 {
+    pub bytes: [u8; 16],
+}
+
+#[derive(Copy, Clone)]
+pub struct Uint160 {
+    pub bytes: [u8; 20],
+}
+
+#[derive(Copy, Clone)]
+pub struct Uint256 {
+    pub bytes: [u8; 32],
+}
+
+type EtherValue = Uint128;
+type Address = Uint160;
+type StorageKey = Uint256;
+type StorageValue = Uint256;
+type Hash = Uint256;
+
+trait EwasmType {
+    fn new() -> Self;
```

Comment:
If we have the default trait, then this is not needed, but you can use `Address::default()`